documentation/source/newsfragments/2490.removal.rst

```diff
@@ -0,0 +1,2 @@
+Setting values on handles using the ``dut.handle = value`` syntax is deprecated.
+Instead use the ``handle.value = value`` or ``handle <= value`` syntax.
```

Comment:
Can you do a follow-up to mention `handle[i] = val` syntax being deprecated too?

Reply:
Sure.